client/components/site-title/index.jsx

```diff
@@ -43,8 +46,22 @@ class SiteTitleControl extends React.Component {
 		this.props.onChange( { blogname, blogdescription } );
 	};
 
+	renderValidation = blogname => {
+		const { translate } = this.props;
+		const isError = ! blogname;
+
+		return <FormInputValidation isError={ isError } text={ translate( 'Required field.' ) } />;
```

Comment:
Hi! I've found a possible matching string that has already been translated **40** times:
`translate( 'Required Fields' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).